tests/phpunit/tests/html-api/wpHtmlTagProcessor.php

```diff
@@ -51,6 +51,54 @@ public function test_get_tag_returns_open_tag_name() {
 		$this->assertSame( 'DIV', $p->get_tag(), 'Accessing an existing tag name did not return "div"' );
 	}
 
+	/**
+	 * @ticket 58009
+	 *
+	 * @covers WP_HTML_Tag_Processor::has_self_closing_flag()
+	 *
+	 * @dataProvider data_has_self_closing_flag
+	 *
+	 * @param string $html input HTML whose first tag might contain the self-closing flag `/`.
+	 * @param bool $flag_is_set whether the input HTML's first tag contains the self-closing flag.
```

Comment:
Maybe start the param descriptions with a capital letter? Might help legibility to delineate the description from the arg name. Also, AFAICS, all the other `@param` descriptions in this file do.

```suggestion
	 * @param string $html Input HTML whose first tag might contain the self-closing flag `/`.
	 * @param bool $flag_is_set Whether the input HTML's first tag contains the self-closing flag.
```